src/renderer/store/index.js

```diff
@@ -0,0 +1,20 @@
+import Vue from 'vue';
+import Vuex from 'vuex';
+
+import persistentState from './storage/PersistentState';
+import projectInfo from './modules/ProjectInfo';
+
+Vue.use(Vuex);
```

Comment:
I'm not sure if this the best place to put that. I'm wondering if this sort of a "main entrypoint"-type of code?

Reply:
`Vue.use` installs the `Vuex` plugin. It calls `plugin.install` which [adds global level functionality to Vue](https://vuejs.org/v2/guide/plugins.html). From my understanding of what it purpose is and all the examples I've looked at, this feels like the best place for it. Before anything else tries to use it or `Vue.store`.